firefox-ios/firefox-ios-tests/Tests/ClientTests/Utils/StoreTestUtility.swift

```diff
@@ -0,0 +1,34 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import Redux
+@testable import Client
+
+protocol StoreTestUtility {
```

Comment:
In my opinion i think we might not need this protocol, i'd just add the ```StoreTestUtilityHelper``` directly as property of the Tester class. Let me know what you think! thanks @cyndichin 😃

Reply:
Thanks for the comment! Hmm, I wanted a way to ensure that any test that uses the store will conform to the protocol, which will throw compiler errors if the user does not have the setup or reset methods and then can use the `StoreTestUtilityHelper` for convenience. Happy to remove if you don't think it's useful though. 

Does it seem unnecessary extra code for you? Do you think we should open up to the team?

Reply:
Ah ok Thank you for clarifying @cyndichin yes that makes sense to me.

Reply:
great, thanks! I'll also present this to the team tomorrow as well so can keep this PR open until then.